src/collatz.js

```diff
@@ -0,0 +1,17 @@
+// If n is even, divide it by 2 to get n / 2. If n is odd, multiply it by 3 and add 1 to obtain 3n + 1. Repeat the process until you reach 1.
+export default function collatz(n){
+
+  let collatzArray = []
```

Comment:
Put a blank line between variable declarations and other logic to increase readability.